src/Propel/Runtime/Map/ColumnMap.php

```diff
@@ -502,6 +502,7 @@ public static function normalizeName($name)
     {
         if (false !== ($pos = strrpos($name, '.'))) {
             $name = substr($name, $pos + 1);
+	     $name = trim($name,"\t\n\r\0\x0B`'");
```

Comment:
in case we strip things like nullbytes etc. should we also strip other control chars?